docs_src/microservices/core/metadata/Ch-Metadata.md

```diff
@@ -132,3 +132,20 @@ Metadata Add a New Device Profile (Step 1 to provisioning a new device)
 ![image](EdgeX_MetadataAddDeviceProfileStep2.png)
 
 ![image](EdgeX_MetadataDeviceStartup.png)
+
+## Configuration Properties
+
+The following are extra configuration parameters specific to the Core-Metadata service. Please refer to the general Configuration documentation for configuration properties common across all services.
```

Comment:
Make general Configuration documentation a link back to the general Configuration documentation